### PR TITLE
Add main content of the page

### PR DESCRIPTION
**Pull request by fernandorpm**
# Milestone 2
> On this milestone we had to create the following components for our website:
> ### Column 1
> - Featured stories
> - Sponsored Insights/Ranking
> ### Column 2
> - Top story
> - My turn
> - Culture & Travel
> - More stories
> - Latest news
> ### Column 3
> - The debate
> - Opinion
> - Sponsored Insights/Ranking
> - Free sign up

## Files to update
- index.html
- styles.css
- README.md

## Extra info
**IMPORTANT: No animations/transitions should be done yet!**
All of the components built were made to work with both desktops, tablets, and smaller mobile devices.

### CODE REVIEW

File: assets/css/global_styles.css [hunk 1]
```diff
@@ -26,7 +26,7 @@ li {
   flex: 0 0 auto;
```
Comment (yokoworks) [inline assets/css/global_styles.css]: [OPTIONAL] If it's not used in HTML, it would be better to be removed.

**Review by yokoworks — CHANGES_REQUESTED** [review]
#  Status: Changes Required♻️ 

**Great job so far, team**👍  

You two have made an amazing job🚀
There are some points to improve before you go ahead.
Let's cover a few details together to make your project better😊 

### Required changes: 
- [x] Use Bootstrap CSS classes everywhere for styling. You use now w100 custom class, it's not a bad way, but in this project, you are supposed to use Bootstrap as possible as you can.
- [x] Each title uses a heading(hx) tag and have a link
- [x] Use section and article tag where they are better to be used for [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)
- [ ] Use GitHub Flow but Re-use the same Pull-Request for the 3 milestones (don't merge after milestone 1 or 2 completion)
- [x] Fix the whole width to 100% of the window. Now it looks over 100% width with an extra margin like the below image
- [x] Icons are not aligned like the below image. They should be set in the correct position.
Artboard


Please request another review once you are ready👍

### With all my respect 🍀

Yoko Saka

**Comment by fernandorpm** [comment]
## For the next reviewer
Unfortunately, we have already merged Milestone 1 to the project and marked the PR as closed.
Is there a way to revert it? If yes, how? If not, can you take it into account?

Thanks in advance

**Review by jubaan — CHANGES_REQUESTED** [review]
[![Microverse][microverse-shield]][microverse-url]
<div>
<h1 align="center"> ⚠️ 🛠️ ⚠️ 🛠️ Changes Required 🛠️ ⚠️ 🛠️ ⚠️ </h1>
<br />
<p><b>"Fail fast, fail often"</b> and don't let this discourage you, because <b>only by failing do we learn to succeed</b>.</p>
<details open="open">
  <summary>Review Summary</summary>
  <ol>
    <li><a href="#congrats"> 💯 Awesome work 👏🏼 👏🏼 </a></li>
    <li><a href="#mandatory"> ⚠️ Things to Improve </a> 🚨 🚨 </li>
    <li><a href="#optional"> 💡Nice to have and other comments ✍🏼 ✍🏼 </a></li>
    <li><a href="#contact"> 🧔🏼 About your Reviewer </a> (you don't know it... but we are just automated bots 🤖) </li>
    <li><a href="#help"> 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼 </a></li>
  </ol>
</details>
</div>

<h3 id="congrats"> 1. :100: Awesome work</h3>
<details close="close">
<summary> [click to open] Big or little achievements, but achievements at the end. <b>Feel proud that you're learning and improving everyday.</b></summary>



</details>

<hr>
<h3 id="mandatory"> 2. ⚠️ Things to Improve (or the To-Do list to get the project approved) </h3> 
<details close="close">
<summary> [click to open] Working on these observations will help you advance and get closer to the goal 🎯, postpone them or do them half-ways and they'll haunt you 👻 👻... FOREVER!! (It will feel like that, trust me). </summary>


- [ ] Prefer Boostrap classes for CSS rules that have a Bootstrap counterpart. Review the CSS file.
- [ ] The category element of the images is still not correctly displayed
image


</details>
<p><b>NOTE</b>This list is or could be divided in what is show in this comment and inline comments posted below.<p>
<hr>

<h3 id="optional"> 3. 💡 Nice to have and other comments </h3>
<details close="close">
<summary> [click to open] </h3></summary>


- [x] To use the same PR just reopen it from GitHub PR section of the project.


</details>

<h3 id="contact"> 4. 🧔🏼 About your Reviewer </h3>
<details close="close">
<summary> [click to open] Say "hello world!" on Slack: @jubaan</summary>
<br />
<div align="center">
  <a src="https://www.github.com/jubaan">
    Julio Añoveros
  </a>
  <p>
    Julio Añoveros | @jubaan</br></br>
    <b>Full-stack Web Developer</b> and</br>
    🤖 Code Reviewer at <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a>:rocket:
  </p>
</div>
</details>

<h3 id="help"> 5. 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼</h3>
<details close="close">
  <summary> [click to open] <b>Feel stuck?</b> </summary>
  <br>
  <p> First breath... just do it. Take 15-20 min away from the computer, stretch, meditate, do some exercise, drink water, or do whatever is necessary to distract your mind from the obstacle.</p>
  <p><b>Sometimes you just need to get away to see things with a new perspective.</b></p>
</details>

<details close="close">
  <summary> [click to open] <b>Need help?</b> </summary>
  <br>
  <p> Send me a DM in <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> Students Slack, you'll find me as @jubaan. If I'm not available there are other Code Reviewers you can contact. They'll help as well.</p>
  <p>Remember that <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> has other channels where you can get an answer to your questions:</p>
  <ol>
    <li>The <b>students channel 🧑‍🎓 </b> in slack,</li>
    <li>your <b>teammates 👥 </b>,</li>
    <li>your <b>mentor 👨🏼 👩🏼 </b>, and</li>
    <li><b>Student Success 🏫 </b>.</li>
  </ol>
  <p><b> DON'T hesitate to ask! </b></p>
</details>
<p>If all of the above doesn't help... get a <a href="https://rubberduckdebugging.com/">rubber duck</a> ... it would be great to own one 😄. </p>

<h2 align="center"> Keep up the great work :muscle: and Happy Coding :smiley: :computer:!! </h1>

[microverse-shield]: https://img.shields.io/badge/-microverse-%236F23FF?style=for-the-badge
[microverse-url]: https://www.microverse.org/?grsf=kfm2ej
[bootstrap-shield]: https://img.shields.io/badge/Bootstrap-white?style=for-the-badge&logo=bootstrap
[bootstrap-url]: https://www.microverse.org/?grsf=kfm2ej

**Comment by fernandorpm** [comment]
## For the next reviewer
Unfortunately, we have already merged Milestone 1 to the project and marked the PR as closed.
Is there a way to revert it? If yes, how? If not, can you take it into account?

Thanks in advance

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @fernandorpm  & @jaarkira**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] you should never use !important tags in your CSS as they can break your cascading order and make your code harder to read. if you are having trouble applying your css attributes to a certain element, please make sure to read this guide about CSS specificity https://www.w3schools.com/css/css_specificity.asp.

 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @fernandorpm :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Make sure that your navbar looks like origninal on mobile screens

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-03-02 15-42-46|Screenshot from 2021-03-02 15-43-17 |

:point_right: Could you see the difference :question: :question: :question: 

- [x] Remove all the `!important` as the previous reviewer pointed it out!
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @fernandorpm and @jaarkira  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

# [Optional] :yellow_circle: 
- [x] Fix the following links in your readme file:
  - [x] Issues link
  - [ ] License link
  - [x] Fernando R P Marques 's linkedin profile link
 - [x] Work on the screenshot, add the actual one. And fix whatever is breaking.
 
 :warning:  The above requirements are optional, you can even choose to them on the last milestone. :warning: 

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**